src/JuMP.jl

```diff
@@ -738,5 +729,19 @@ include("print.jl")
 # Nonlinear-specific code
 include("nlp.jl")
 
+
+typealias _getTypes Union{NonlinearExpression,NonlinearParameter, AffExpr}
+getValue{T<:_getTypes}(arr::Array{T}) = map(getValue, arr)
```

Comment:
I think I would prefer it if this was ``T <: JuMPTypes`` (with ``NonlinearExpression <: JuMPTypes`` etc. elsewhere). I think that would also help with the problem with more specific behavior for ``Array{Variable}`` mentioned above (since this would be a stricter type signature).